src/qt/rpcconsole.cpp

```diff
@@ -702,7 +702,7 @@ void RPCConsole::addWallet(WalletModel * const walletModel)
         // First wallet added, set to default so long as the window isn't presently visible (and potentially in use)
         ui->WalletSelector->setCurrentIndex(1);
     }
-    if (ui->WalletSelector->count() > 2) {
```

Comment:
Before the wallet selector would be visible with only 2 wallets. However I think it makes sense to show the selector even with 1 wallet only because otherwise you can't decide to execute `unloadwallet` with or without a wallet.

Reply:
Moved this to #15150.